docs/source/format/Columnar.rst

```diff
@@ -1045,7 +1045,10 @@ file. Further more, it is invalid to have more than one **non-delta**
 dictionary batch per dictionary ID (i.e. dictionary replacement is not
 supported). Delta dictionaries are applied in the order they appear in
 the file footer. We recommend the ".arrow" extension for files created with
-this format.
+this format. Note that files created with this format are sometimes called
+"Feather V2", a name derived from "Feather (V1)", which was a proof of concept
```

Comment:
Should we note that sometimes they still use the ".feather" extension?

Reply:
I added a note about the `.feather` extension in ac36ed3c3a021d38b2eb67511ed2ce490fb54368.
I think either is fine and I will defer to the decision of the maintainers.